### DISCUSSION TITLE
[Suggestion] Gravity override

### DISCUSSION THREAD

**Discussion by cloudrambler** [tsunamayo/Starship-EVO#4095] [Suggestions]
The default interaction of multiple gravity generators is to add their effect when the overlap. That arrangement has it's uses.

Another useful arrangement would be to have a unified gravity field, so that overlaps are ignored. This would make it easier to fill a ship with generators, as it would not matter if some overlapped.

The final arrangement would be for certain gravity generators to have exclusive influence, so that their field is not affected by any overlapping fields, with those others being ignored in that region. This would be useful for moving platforms that pass through regions of alternate gravity orientation.

So I suggest the gravity generators have two toggle boxes; one for additive and one for exclusive, with the default being unified field.

**Comment by tsunamayo** [comment]
hum smart idea indeed...
